packages/snap/README.md

```diff
@@ -0,0 +1,29 @@
+
+## Introducing Saferoot: Safeguard your crypto and NFTs
+
+[Saferoot](https://www.saferoot.io/) is a pioneering **threat mitigation layer**, providing **real-time protection** for vulnerable digital assets. Our non-custodial solution intercepts dangerous transactions before being committed on-chain, redirecting pre-authorized assets to a secure **backup wallet.**
+
+### Why Choose Saferoot?
+
+In just the first quarter of 2022, losses in the Web3 ecosystem due to hacks and fraud amounted to $1.23 billion—a staggering 695% increase from the previous year. The FTC reported that over 46,000 people lost more than $1 billion to crypto scams in 2021 alone. Saferoot stands as a robust countermeasure to this escalating problem.
+
+Discover our main dApp at [Saferoot](https://app.saferoot.io).
+Learn more about Saferoot in our [Docs](https://docs.saferoot.io/overview/what-is-saferoot).
+
+### Saferoot & MetaMask Snaps
+
+**Current Features:**
++ Interactive prompts assist you when you need more information.
+
+**Upcoming Features:**
++ Multi-asset approval support is on the horizon.
+
+### Run Saferoot Snap locally
+
+- `yarn install`
+- `yarn start`
+
+### Notes
+
+- MetaMask Flask is currently compatible only with desktop browsers like Chrome and Firefox.
+- For optimal performance, install Flask in a fresh browser profile or deactivate any existing versions of MetaMask. Running multiple instances of MetaMask in the same browser profile can disrupt dApp interactions.
```

Comment:
Why the README add?

Reply:
Copied off the main readme.md
Necessary so it can show these details on the npm package [link](https://www.npmjs.com/package/saferoot-metamask-snap)